src/ebay-badge/README.md

```diff
@@ -20,7 +20,7 @@ import '@ebay/skin/badge'
 
 ## ...or using SCSS/CSS
 ```jsx harmony
-import '@ebay/skin/dist/badge/ds6/badge.css'
+import '@ebay/skin/dist/badge/badge.css'
```

Comment:
Is the `dist` needed? On less 13 we import using `@import "~@ebay/skin/badge.css";` (this should be valid for all the others component as well)

Reply:
Sounds good, I was not aware that most `import '@ebay/skin/dist/XYZ/XYZ.css' are also available via `import '@ebay/skin/XYZ.css'. Normally, I would prefer to use webpack import `import '@ebay/skin/dist/XYZ'. Will update accordingly.